button.js

```diff
@@ -2,7 +2,7 @@ function changeColor() {
   var blueDiv = document.getElementById("changeable");
   // Use the Mozilla Developer Network resource to find the right way to change
```

Comment:
You can remove any comments that don't serve a specific purpose in your finished code.

Reply:
I will remove the comments right away. Thanks for the feedback